typings/love.physics/types/Joint.d.ts

```diff
@@ -60,11 +60,13 @@ declare module "love.physics" {
         /**
          * Gets the reaction force on Body 2 at the joint anchor.
          *
+         * @param invdt How long the force applies. Usually the inverse time step or 1/dt.
+         * 
```

Comment:
Could I request line 64 be removed? Other than that happy to merge this change

Reply:
Sure, but I don't know how to make changes to PR when I deleted my fork